src/lib.rs

```diff
@@ -624,11 +633,14 @@ pub async fn transfer_assets(
     String, // bech32m compressed sten consignment
     String, // base64 bitcoin encoded psbt
     String, // json
+    String,
```

Comment:
It would be great to comment those new strings, change, previous_utxo and new_utxo